tools/codegen/config.yml

```
@@ -461,6 +461,8 @@ resources:
     version_header: application/vnd.atlas.2023-01-01+json
     schema:
       ignores: ["links"]
+      aliases:
+        username: dbUser
```

```suggestion
        username: dbUser # alias is only added to illustrate full aliasing support (username is both a path param and present in request body)
```
Can be helpful for future reference